addons/forum-time-zones/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Use local timezone on forums",
-  "description": "Converts the post dates to your time zone. Uses ScratchDB for information.",
+  "name": "Local timezone on forums",
+  "description": "Converts the post dates on the forums to use your local time zone. Uses ScratchDB for information.",
```

Comment:
`Converts the post dates on the forums to use your local time zone.` I'm not sure if this English sentence is grammatically correct. Someone native in English help?

Reply:
`Converts forum post date and times to your local time zone`

Reply:
"Use" and "local" have to be there for reasons I mentioned in on a comment on Apple's review.